src/Menu/Menu.stories.tsx

```diff
@@ -46,7 +46,7 @@ storiesOf('Components/Menu', module)
                 </Menu>
                 <br />
                 <p>Small, Custom Color, & Custom Position</p>
```

Comment:
This `p` should prolly be a `Typography`

Reply:
That `&` may need to be an `&amp;`

Reply:
I'll update all the instances in the story using the semantic tag.